CODEOWNERS

```diff
@@ -206,6 +206,7 @@ Kconfig*                                  @tejlmand
 /tests/lib/date_time/                     @trantanen @tokangas
 /tests/lib/edge_impulse/                  @pdunaj @MarekPieta
 /tests/lib/hw_unique_key*/                @oyvindronningstad @Vge0rge
+/tests/lib/location/                      @trantanen
```

Comment:
Could we match this with `/lib/location/`:
`@trantanen @hiltunent @jhirsi @tokangas`

Reply:
This has been discussed with Location library owners so feel free to get this merged.